src/probnum/filtsmooth/__init__.py

```diff
@@ -1,8 +1,19 @@
 """Bayesian Filtering and Smoothing."""
 
-from .bayesfiltsmooth import *
-from .filtsmoothposterior import *
-from .gaussfiltsmooth import *
+from .bayesfiltsmooth import BayesFiltSmooth
+from .filtsmoothposterior import FiltSmoothPosterior
+from .gaussfiltsmooth import (
+    ContinuousEKFComponent,
+    ContinuousUKFComponent,
+    DiscreteEKFComponent,
+    DiscreteUKFComponent,
+    FixedPointStopping,
+    IteratedKalman,
+    Kalman,
+    KalmanPosterior,
+    StoppingCriterion,
+    UnscentedTransform,
+)
 
```

Comment:
Shouldn't we also import the `statespace` module here?

Reply:
is this how it works? I thought having it separate would be sufficient...

Reply:
Before merging, you might want to reconsider that. I think, it works as it is, but I see no reason why in the `filtsmooth.__init__.py` we should explicitly import `gaussfiltsmooth` but not `statespace`.